guessGame.html

```diff
@@ -1,45 +1,68 @@
 <script>
-alert("The object of this guessing is to guess how many times you will be able to correctly guess a random number between 1 and 5 within 5 tries. You will be asked your initial guess first, and then you'll guess a random number between 1 and 5 5 times. You will then recieve a message informing you of your win or your loss!");
-
-var counter = 0;
-var initialGuess = prompt("If you have 5 guesses, how many times can you correctly guess a random number between 1 and 5?");
-
-Number(initialGuess);
-
-while(isNaN(initialGuess) || initialGuess === "" || initialGuess > 5) {
-    alert("You can only guess a number between 0 and 5! Please guess again!");
-    initialGuess = prompt("If you have 5 guesses, how many times can you correctly guess a random number between 1 and 5?");
-
-Number(initialGuess);
-}
-for (i = 0; i < 5; i++) {
-  var guess = prompt("Guess a number!");
-  Number(guess);
-  while (isNaN(guess) || guess === "" || guess > 5 || guess < 1) {
-    alert("You can only guess a number between 1 and 5! Please guess again!");
-    guess = prompt("Guess a number!");
-    Number(guess);
-  }
-
-  var answer = Math.floor(Math.random() * 5 + 1);
-  var message;
+  alert("The object of this guessing is to guess how many times you will be able to correctly guess a random number between 1 and 5 within 5 tries. You will be asked your initial guess first, and then you'll guess a random number between 1 and 5 5 times. You will then recieve a message informing you of your win or your loss!");
 
-  if (guess == answer) {
-    message = "Nice! You guessed correctly!";
-    counter++;
+  
+  var initialGuess = prompt("If you have 5 guesses, how many times can you correctly guess a random number between 1 and 5?");
+  
+  if (initialGuess === "q") {
+    alert("Goodbye!");
   }
+  
   else {
-    message = "Wrong! You guessed incorrectly! The answer was " + answer + "! Too bad, so sad...";
-  }
 
-  alert(message);
-}
+    Number(initialGuess);
+
+    while(isNaN(initialGuess) || initialGuess === "" || initialGuess > 5) {
+      alert("You can only guess a number between 0 and 5! Please guess again!");
+      initialGuess = prompt("If you have 5 guesses, how many times can you correctly guess a random number between 1 and 5?");
+      Number(initialGuess);
+    }
+
+    var counter = 0,
+        exit = false;
+
+    for (i = 0; i < 5; i++) {
+      var guess = prompt("Guess a number!");
+      if (guess === "q") {
+        i = 6; 
+        alert("Goodbye!");
+        exit = true;
+      }
+      else {
+        Number(guess);
+        while (isNaN(guess) || guess === "" || guess > 5 || guess < 1) {
```

Comment:
this while looks just like the while above. can you put it in a function and use the function in both places?